src/basic/json.c

```diff
@@ -24,10 +24,10 @@
 #include "terminal-util.h"
 #include "utf8.h"
 
-/* Refuse putting together variants with a larger depth than 16K by default (as a protection against overflowing stacks
+/* Refuse putting together variants with a larger depth than 4K by default (as a protection against overflowing stacks
  * if code processes JSON objects recursively. Note that we store the depth in an uint16_t, hence make sure this
  * remains under 2^16. */
```

Comment:
could you add a comment here saying that 16K was discovered to be too high on llvm/x86-64?

Reply:
OK. Done.